recipe/activate.sh

```diff
@@ -16,7 +16,7 @@ if [ "${CONDA_BUILD:-0}" = "1" ]; then
     CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${BUILD_PREFIX}/${targetsDir}/lib"
     CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${BUILD_PREFIX}/${targetsDir}/lib/stubs"
     # Needed to fix cross compilation
-    export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_FIND_ROOT_PATH=$PREFIX;$BUILD_PREFIX/$HOST/sysroot;$BUILD_PREFIX/${targetsDir}"
+    export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_FIND_ROOT_PATH=$PREFIX;$BUILD_PREFIX/$HOST/sysroot;$PREFIX/${targetsDir};$BUILD_PREFIX/${targetsDir}"
```

Comment:
Do we want this to come before `sysroot` in the search order?

Reply:
I don't expect we do. The `$BUILD_PREFIX/$HOST/sysroot` contents should be compiler toolchain bits and if we have a conflict with `$PREFIX/${targetsDir}` I expect that `sysroot` is the correct one.